rust/arrow/src/compute/kernels/aggregate.rs

```diff
@@ -27,40 +27,51 @@ pub fn min<T>(array: &PrimitiveArray<T>) -> Option<T::Native>
 where
     T: ArrowNumericType,
 {
-    min_max_helper(array, |a, b| a < b)
+    min_max_helper(array, |a, b| a > b)
 }
 
 /// Returns the maximum value in the array, according to the natural order.
 pub fn max<T>(array: &PrimitiveArray<T>) -> Option<T::Native>
 where
     T: ArrowNumericType,
 {
-    min_max_helper(array, |a, b| a > b)
+    min_max_helper(array, |a, b| a < b)
```

Comment:
was the ordering here originally incorrect?

Reply:
No, but a change in the implementation of `min_max_helper` made it require the opposite. We could probably change it to keep this unchanged, but since this is so specific to this module, I did not bother.

Reply:
No worries, happy with the change